companies/tibush.json

```
@@ -0,0 +1,30 @@
+{
+    "slug": "tibush",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "entertainment",
+        "social media"
+    ],
+    "name": "tibush GmbH",
+    "runs": [
+        "SpaceHey"
+    ],
+    "address": "Oberhaldenweg 32\n72793 Pfullingen\nDeutschland",
```

This isn't a great solution but if the `relevant-countries` cover more than the country where the company is based, we write the country in English for better deliverability.

```suggestion
    "address": "Oberhaldenweg 32\n72793 Pfullingen\nGermany",
```